payment/app.py

```diff
@@ -72,28 +72,31 @@ def pay():
 def watch(service_id):
     body = request.get_json()
     body['service_id'] = service_id
+    order_id = body.pop('order_id')
```

Comment:
the same comment as in marketplace, I think 'watcher id' or 'id' is more appropriate than order_id

Reply:
id is sort of a "saved word" in python (id()), and watcher_id is his address, so I will it leave it as it